### PR TITLE
feat(header-component): apply new styles

### PR DESCRIPTION
**Pull request by fcoronelmakingsense**
- Add new styles and layout to user right menu.

CM: Profile
image

CM: User
image

Demo
image

Enterprise: suscribers
image

Highvolume: emails
image

Standard: suscribers
image

Prepaid
image

Free
image

#Checklist

- [] Check if there is a new color.
- [] Create the color variable in the _colors.scss file.
- [] Follow the color nomenclature.
- [] Create the color class with its variable.
- [] Create the color component in the html, show the name, color class and its exadecimal.

### CODE REVIEW

File: assets/scss/templates/_header.scss [hunk 1]
```diff
@@ -368,15 +368,20 @@
...
 
     .user-plan {
       color: $dp-color-green;
       font-size: 11px;
       text-transform: uppercase;
+      margin-top: 10px;
+    }
+
+    .user-plan-enabled {
+      font-size: 11px;
```
Comment (cbernat) [inline assets/scss/templates/_header.scss:381]: I think we have a variable for font sizes...

File: assets/templates/header-component.html [hunk 2]
```diff
@@ -152,18 +152,178 @@
                         ><span class="email">gbaamonde@makingsense.com</span>
                       </p>
                     </header>
-                    <div class="user-plan--container m-t-18">
+                    <!-- <div class="user-plan--container m-t-18">
```
Comment (cbernat) [inline assets/templates/header-component.html:155]: Why do you have all this markup commented? cannot it be left in another page with all examples working?

File: assets/scss/templates/_header.scss (hunk 1 above)
Comment (fcoronelmakingsense) [reply assets/scss/templates/_header.scss]: changed

File: assets/templates/header-component.html (hunk 2 above)
Comment (fcoronelmakingsense) [reply assets/templates/header-component.html]: changed

**Pull request merged**